src/OmniSharp.Http/Host.cs

```diff
@@ -44,7 +45,7 @@ public void Start()
                     serviceCollection.AddSingleton(_environment);
                     serviceCollection.AddSingleton(_sharedTextWriter);
                     serviceCollection.AddSingleton(NullEventEmitter.Instance);
-                    serviceCollection.AddSingleton(_pluginAssemblies);
+                    serviceCollection.AddSingleton(_commandLinePlugins);
```

Comment:
Registering an `IEnumerable<string>` as a service seems like a code smell to me.

Reply:
@JoeRobich , are you proposing having a dedicated class containing the list?

Reply:
I think registering a `PluginAssemblies` here to represent the Plugins the environment was initialized with would make sense.

Reply:
@JoeRobich Reworked this part - please take another look. Thanks!